js/views/components/ModeratorsStatus.js

```diff
@@ -42,9 +42,10 @@ export default class extends BaseVw {
       this.$el.html(t({
         ...this.getState(),
       }));
+      if (this.spinnerTimeout) clearTimeout(this.spinnerTimeout);
```

Comment:
As an fyi, clearTimeout handles pretty much anything being passed to it, so you really don't need that if.